app-modules/ai/src/Jobs/CloneAiThread.php

```diff
@@ -82,6 +89,15 @@ public function handle(): void
         $threadReplica->locked_at = now();
         $threadReplica->save();
 
+        if (Feature::active('ai_utilization')) {
+            $this->thread->cloned_count = $this->thread->cloned_count + 1;
+            $this->thread->save();
+        }
+
+        if (Feature::active('ai_utilization') && Cache::has('special_actions_doughnut_chart')) {
+            Cache::forget('special_actions_doughnut_chart');
```

Comment:
We may want to make this more generic and tied to the clone count. Something like `thread-clone-count`

Reply:
As you added comment to remove cache from that chart, we didn't need this anymore.